src/components/ViewAdmin.vue

```diff
@@ -306,6 +306,36 @@
 				<input v-model="settings['node_binary']" type="text" @change="onChange">
 			</p>
 		</NcSettingsSection>
+		<NcSettingsSection :title="t('recognize', 'Nice value')">
+			<p v-if="nice === undefined">
+				<span class="icon-loading-small" />&nbsp;&nbsp;&nbsp;&nbsp;{{ t('recognize', 'Checking Nice binary') }}
+			</p>
+			<NcNoteCard v-else-if="nice === false">
+				{{ t('recognize', 'Could not find the Nice binary. You may need to set the path to a working binary manually.') }}
+			</NcNoteCard>
+			<p>
+				{{ t('recognize', 'Nice binary path') }}
+			</p>
+			<p>
+				<input v-model="settings['nice_binary']"
+					type="text"
+					:placeholder="t('recognize', 'Nice binary path')"
+					@change="onChange">
+			</p>
+			<p>&nbsp;</p>
+			<p>
+				{{ t('recognize', 'Nice value to set the priority of the Node.js processes. The value can only be from 0 to 19 since the Node.js process runs without superuser privileges.') }}
```

Comment:
I think we should explain what nice does here, too, because people may not be familiar with it.